sys-devel/icecream/icecream-1.3.1.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools systemd tmpfiles
+
+DESCRIPTION='Distributed compiling of C(++) code across several machines; based on distcc'
+HOMEPAGE='https://github.com/icecc/icecream'
+SRC_URI="https://github.com/icecc/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE='GPL-2'
+SLOT='0'
+KEYWORDS="~amd64 ~arm ~hppa ~ppc ~sparc ~x86"
+IUSE='systemd'
+
+DEPEND='
+	acct-user/icecream
+	acct-group/icecream
+	sys-libs/libcap-ng
+	app-text/docbook2X
+	app-arch/zstd
+'
+RDEPEND="
+	${DEPEND}
+	dev-util/shadowman
+	systemd? ( sys-apps/systemd )
+"
+
+AT_NOELIBTOOLIZE='yes'
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	econf \
+		--enable-clang-rewrite-includes \
+		--enable-clang-wrappers \
+		--disable-fast-install
+}
+
+src_install() {
+	default
+	find "${D}" -name '*.la' -delete || die
+
+	if use systemd; then
```

Comment:
Just install these [unconditionally](https://projects.gentoo.org/qa/policy-guide/installed-files.html#pg0301). Drop the systemd useflag.